### PR TITLE
Select:fix Select box jitter when the window is zoomed bug（对应issue：#21142）

### PR DESCRIPTION
**Pull request by caoshining**
移出动态监听致使持续重绘 导致的高度持续更改，解决对应问题 对应 #21142 

Please make sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow Element's contributing guide ([中文](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.zh-CN.md) | [English](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.en-US.md) | [Español](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [x] Make sure you are merging your commits to `dev` branch.
* [x] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

**Comment by caoshining** [comment]
Hi! Element UI team! Thanks for creating such an excellent UI library.

Window zoom jitters when multiple selection occurs，The component is constantly updating its calculations，I think it is the jitter caused by the real-time calculation of the setting height within the component

File: packages/select/src/select.vue [hunk 1]
```diff
@@ -795,7 +795,6 @@
 
       handleResize() {
         this.resetInputWidth();
-        if (this.multiple) this.resetInputHeight();
```
Comment (cs1707) [inline packages/select/src/select.vue:798]: 这行不能删

Comment (caoshining) [reply packages/select/src/select.vue:798]: 但是一直重新计算的问题是这个引起的 能解决下吗？

Comment (caoshining) [reply packages/select/src/select.vue:798]: @cs1707 老哥 你看348 行已经监听value来重置高度了，handleResize的绑定那就会无限触发，只要我新增或者移除一个选项，你可以对比下看下348行 已经在重算和重置高度了 那你这行监听大小变化不久一直会触发了？无限循环了 只用value更改的时候重置就够了 你不信你可以跑一下我改的这个代码试下 解决了了这个issue ，麻烦看下 谢谢

**Pull request closed without merging**